mindsdb/streams/redis/redis_stream.py

```diff
@@ -0,0 +1,56 @@
+import os
+import json
+from threading import Thread, Event
+import walrus
+from mindsdb.utilities.log import log
+from mindsdb_native import Predictor
```

Comment:
you should not use `mindsdb_native` directly, make prediction through `naitve_interface`

Reply:
Done